voice/join-outbound-calls.py

```diff
@@ -3,27 +3,23 @@
 
 app = Flask(__name__)
 
+
```

Comment:
Missing where the variables here come from - supposed to edit ahead of time? Maybe we could use dotenv? :)